.github/workflows/tests.yaml

```diff
@@ -44,10 +44,14 @@ jobs:
         run: |
           sudo apt update
           sudo apt install python3.11-dev -y
+          pip install -U pip uv
```

Comment:
here maybe we could also benefit from the Python installation with `uv install python 3.11`, as well as installing `uv` from the standalone installer instead of `pip` WDYT? reference at https://docs.astral.sh/uv/getting-started/installation/#standalone-installer